### PR TITLE
Update steering council membership

### PR DESCRIPTION
**Pull request by jhamman**
cc @pangeo-data/pangeo-steering-council

### CODE REVIEW

**Comment by jhamman** [comment]
I've put this up a bit prematurely. We are missing one procedural measure. Will convert this from a draft when ready.

**Comment by chiaral** [comment]
Hi @jhamman - very happy to see all these new additions. 
I decided to step down as well. I can modify the PR or maybe it's just easier you update it accordingly.
Thanks!

**Review by rabernat — APPROVED** [review]

**Review by rsignell-usgs — COMMENTED** [review]
looks good to me!
oops, will approve now.

**Review by rsignell-usgs — APPROVED** [review]

**Review by cgentemann — APPROVED** [review]

**Review by chiaral — APPROVED** [review]

File: steering_council_membership.md
Comment (rabernat) [inline steering_council_membership.md:10]: Chelle has already been on the SC...we just didn't update this page.

Comment (jhamman) [reply steering_council_membership.md:10]: The list was reordered slightly but she was in fact already on the list.

File: steering_council_membership.md
Comment (niallrobinson) [inline steering_council_membership.md:3]: All looks good to me

**Review by niallrobinson — APPROVED** [review]
Looks good to me

**Comment by rabernat** [comment]
I think this needs approval of the _current steering council_, including those who are stepping down, so I have added @amanda-tan, @RPrudden, and @guillaumeeb to the approvers list.

**Review by rabernat — APPROVED** [review]

**Review by guillaumeeb — APPROVED** [review]
A little sad, but this needs more time than I have, and I'll stay around.

**Comment by rabernat** [comment]
> A little sad, but this needs more time than I have, and I'll stay around.

😢  Guillaume you will always be part of the Pangeo family!

**Comment by rabernat** [comment]
@RPrudden & @amanda-tan, would you mind clicking "approve" so that we can finalize this?

**Review by RPrudden — APPROVED** [review]

**Review by amanda-tan — APPROVED** [review]

**Comment by rabernat** [comment]
Great! This is done! Thanks everyone! I'll merge this and we will move forward with announcing the new steering council membership.

**Pull request merged**
